.lintrunner.toml

```diff
@@ -72,6 +73,34 @@ command = [
 ]
 is_formatter = true
 
+[[linter]]
+code = 'IWYU'
+include_patterns = [
+]
+exclude_patterns = [
+]
+init_command = [
+    'python3',
+    'tools/linter/adapters/s3_init.py',
+    '--config-json=tools/linter/adapters/s3_init_config.json',
+    '--linter=iwyu',
+    '--dry-run={{DRYRUN}}',
+    '--output-dir=.lintbin',
+    '--output-name=iwyu',
+]
```

Comment:
@suo can you help with `lintrunner init`? I need to download two files `include-what-you-use` and `iwyu_tool`.

Reply:
@suo PTAL